components/PlantImage.js

```diff
@@ -0,0 +1,74 @@
+import Image from "next/image";
+import Link from "next/link";
+import styled from "styled-components";
+import SvgIcon, { icons } from "@/components/SvgIcon";
+import { useRouter } from "next/router";
+
+const PlantImageContainer = styled.div`
+  position: relative;
+`;
+
+const StyledImage = styled(Image)`
+  border-radius: 0.5rem;
+  position: relative;
```

Comment:
not a hundred percent sure if this position property makes sense for an Image, i think it shouldn't be necessary, but i cant test it locally, because i can't fetch the data

Reply:
Refactored this component internally, including positioning.